### ISSUE TITLE
Create custom style for number fields

### ISSUE THREAD

**Issue by crawfoal** [crawfoal/greensteps#31]
I'd like to have number fields that have big increment and decrement buttons and has the default spinners hidden.

**Comment by i-hardy** [comment]
@crawfoal Do you mean something like [this](https://codepen.io/komarovdesign/pen/PPRbgb)? I'd be happy to take this on.

**Comment by crawfoal** [comment]
Yep, kind of like that 😃  For me that one is a little hard on the eyes - the borders are hard to see and I think that the + and - icons are too small.

For Sass, the app is using Bourbon and Bitters. The Bitters styles are in `stylesheets/base`. I'm still deciding on conventions for Sass stuff, but I like the idea of defining component styles in mixins and having classes and ids describe what the element is rather than how it looks. If you look through `stylesheets`, you'll notice not everything follows this. And I don't like the organization of certain things in there, so any suggestions are welcome 😃

**Comment by i-hardy** [comment]
Okay awesome, I'll see what I can come up with.

**Comment by crawfoal** [comment]
Hi @i-hardy, just checking in. Are you still planning on finishing this? Is there anything I can help you with?
